docs/M365/MDO/MDOThreatPolicyChecker.md

```diff
@@ -0,0 +1,69 @@
+# MDOThreatPolicyChecker
+This script checks which Microsoft Defender for Office 365 and Exchange Online Protection threat policies cover a particular user, including anti-malware, anti-phishing, inbound and outbound anti-spam, as well as Safe Attachments and Safe Links policies in case these are licensed for your tenant. In addition, the script can check for threat policies that have inclusion and/or exclusion settings that may be redundant or confusing and lead to missed coverage of users or coverage by an unexpected threat policy.
+
+## Download
+Download the latest release: [MDOThreatPolicyChecker.ps1](https://github.com/microsoft/CSS-Exchange/releases/latest/download/MDOThreatPolicyChecker.ps1)
+
```

Comment:
Let's move this to the top. Should be similar to our other main scripts like Health Checker and Exchange Extended Protection Managment Script. 

![image](https://github.com/microsoft/CSS-Exchange/assets/22776718/be19bc99-fc51-490f-9005-4a5f2ef6d75f)